pkgs/os-specific/linux/lsb-release/default.nix

```diff
@@ -16,7 +16,14 @@ stdenv.mkDerivation rec {
 
   installFlags = [ "prefix=$(out)" ];
 
-  buildInputs = [ perl getopt ];
+  nativeBuildInputs  = [ makeWrapper coreutils getopt ];
+
+  buildInputs = [ perl ];
```

Comment:
perl is native (runs when building), coreutils and getopt are target (run where this package is supposed to run).

Reply:
Whoops! Fixed, thanks!